internal/datacoord/services.go

```diff
@@ -1021,19 +1021,21 @@ func (s *Server) Import(ctx context.Context, itr *datapb.ImportTaskRequest) (*da
 		log.Error("import failed as all dataNodes are offline")
 		return resp, nil
 	}
+
 	avaNodes := getDiff(nodes, itr.GetWorkingNodes())
 	if len(avaNodes) > 0 {
 		// If there exists available DataNodes, pick one at random.
 		dnID := avaNodes[rand.Intn(len(avaNodes))]
```

Comment:
nit: resp.DatanodeId = avaNodes[rand.Intn(len(avaNodes))]

and use  `resp.GetDatanodeId()` instead of `dnID` in the code below.